Formula/utica.rb

```diff
@@ -1,17 +1,14 @@
 class Utica < Formula
   desc "Decentralized dependency manager for Cocoa"
   homepage "https://github.com/Interfere/Utica"
-  url "https://github.com/Interfere/Utica.git",
-      tag:      "0.40.0-pre",
-      revision: "1fc1fba5a4d34117b423d475f594ea433b8efd70",
-      shallow:  false
+  url "https://github.com/Interfere/Utica/releases/download/0.40.0-pre/Utica.zip"
```

Comment:
I would recommend using the brew bottle instead.
see https://docs.brew.sh/Bottles

Reply:
@Interfere Updated, thanks for your suggestion!